cockroachcloud/client-certs-dedicated.md

```diff
@@ -0,0 +1,402 @@
+---
+title: Manage Certificate Authentication for SQL Clients
+summary: procedures for managing client certificates for dedicated clusters
+toc: true
+docs_area: manage.security
+---
+
+SQL clients may authenticate to {{ site.data.products.dedicated }} clusters using public key infrastructure (PKI) security certificates as an alternative to authenticating using a username and password or using [Cluster Single Sign-on (SSO) using CockroachDB Cloud Console](cloud-sso-sql.html) or [Cluster Single Sign-on (SSO) using JSON web tokens (JWT)](../{{site.versions["stable"]}}/sso-sql.html). This page describes how to administer the cluster's certificate authority (CA) certificate and authenticate to a cluster using client certificates.
```

Comment:
SSO using CC Console / CLI is still suggested, so we should present it such that client certs are complimentary to that. That way, customers could use SSO for human users logging in for ad-hoc operations via CLI, and apps could connect via certs (as an alternative to JWT).